foundationdb/src/tuple/mod.rs

```diff
@@ -196,4 +238,39 @@ mod tests {
 
         assert_eq!("string".encode_to_vec(), ("string",).encode_to_vec());
     }
+
+    #[test]
+    fn test_recursive_tuple() {
+        assert_eq!(
+            &("one", ("two", 42)).encode_to_vec(),
+            &[2, 111, 110, 101, 0, 2, 116, 119, 111, 0, 21, 42]
+        );
+        assert_eq!(
+            &("one", ("two", 42, ("three", 33))).encode_to_vec(),
```

Comment:
```
>>> [ord(x) for x in fdb.tuple.pack( ("one", ("two", 42, ("three", 33) ) ) )]
[1, 111, 110, 101, 0, 5, 1, 116, 119, 111, 0, 21, 42, 5, 1, 116, 104, 114, 101, 101, 0, 21, 33, 0, 0]
```

Reply:
Ah, you're right... I should have looked more closely. I'll drop this impl from the change.